src/core/airspacemonitor.cpp

```diff
@@ -1178,9 +1224,17 @@ namespace swift::core
         const CSimulatedAircraft aircraft = this->getAircraftInRangeForCallsign(callsign);
         if (aircraft.hasValidCallsign())
         {
-            // only if we do not have a DB model yet
-            if (!aircraft.getModel().hasValidDbKey())
+            // we do not change manually assigned models (msfs2024)
+            if (!aircraft.getModel().hasValidDbKey() && aircraft.getModelType() != CAircraftModel::TypeManuallySet)
             {
+
+                CLogMessage(this).warning(u"CAirspaceMonitor::reverseLookupModelWithFlightplanData "
```

Comment:
I assume this case happens regularly in normal operation? If so, it should not be logged as warning